packages/api/blocks-api/src/blocks/block.ts

```diff
@@ -53,10 +53,16 @@ export declare type BlockIndexItem = {
 } & BlockIndexData;
 export declare type BlockIndex = Array<BlockIndexItem>;
 
+export interface BlockWarningReport {
+    message: string;
+    severity: string; // TODO: Use WarningSeverity here, as soon as it is in the package and not in the demo (cannot import from demo)
```

Comment:
Maybe copy the enum here and add a TODO to replace the enum instead? This way it would be easier to replace, without having to change types (string to Enum)

Reply:
7f3f076a048d57a61438d1ff5323f0ffdd82ac4a